apps/web/src/scenes/NftDetailPage/NftDetailText.tsx

```diff
@@ -261,23 +415,22 @@ const StyledOwnerAndCreator = styled(HStack)`
 
 const StyledDetailLabel = styled.div<{ horizontalLayout: boolean; navbarHeight: number }>`
   display: block;
-  max-width: 296px;
-  min-width: 296px;
+  max-width: 420px;
+  min-width: 420px;
```

Comment:
looks like this change might affect mobile resposiveness, maybe can use a % if it's a smaller breakpoint?

prod
![CleanShot 2023-11-21 at 18 43 15](https://github.com/gallery-so/gallery/assets/80802871/b1f1bb93-3383-4026-825b-8001c7561096)

dev preview
![CleanShot 2023-11-21 at 18 43 22](https://github.com/gallery-so/gallery/assets/80802871/013b75be-9155-4ded-86fe-6a9be45e27f8)